lib/primitive/test/spec/Cardano/Wallet/Primitive/CollateralSpec.hs

```diff
@@ -698,8 +693,8 @@ spec = do
         describe "generators and shrinkers" $ do
             it "generates values with sufficient coverage" $
                 property prop_genAddress_coverage
-            it "shrink maintains validity" $
-                property prop_simplifyAddress_validAddress
+            -- it "shrink maintains validity" $
+            --     property prop_simplifyAddress_validAddress
```

Comment:
I managed to easily re-enable 
- `prop_simplifyAddress_validAddress`
- `prop_addressType_stake`

Will push. The rest didn't seem as straight-forward, but I think it is fine to leave.